ci-operator/jobs/infra-periodics.yaml

```diff
@@ -1195,9 +1195,7 @@ periodics:
       - --config-dir=./ci-operator/config/
       - --current-release=4.8
       - --future-release=4.9
-      - --confirm
```

Comment:
Huh the tool defaults to `--dry-run=false` to allow us to jsut remove this? That looks wrong.

Reply:
I default it to false not to break the job but I have also to change the `token-path` either way. The `--confirm` argument is still valid but it doesn't affect anything.